lib/datadog/tracing/distributed/baggage.rb

```diff
@@ -125,6 +130,33 @@ def parse_baggage_header(baggage_header)
           end
           baggage
         end
+
+        # Convert baggage items to trace_distributed_tags with 'baggage.' prefix
+        # based on the baggage_tag_keys configuration
+        def build_baggage_tags(baggage)
+          return {} if baggage.empty?
+
+          # Get the configuration for which baggage keys should become span tags
+          baggage_tag_keys = ::Datadog.configuration.tracing.baggage_tag_keys
+          return {} if baggage_tag_keys.empty?
+
+          # If wildcard is specified, use all baggage keys
+          baggage_tag_keys = baggage.keys if baggage_tag_keys == BAGGAGE_TAG_KEYS_MATCH_ALL
+
+          tags = {}
+
+          # Convert configured keys to span tags
```

Comment:
This comment can be removed, because it's simply reading the code below, and it doesn't add value not already present in the code.
```suggestion
```